pkg/kcp/provider/gcp/vpcpeering/state.go

```diff
@@ -63,15 +65,13 @@ func newState(vpcPeeringState vpcpeeringtypes.State,
 		State:              vpcPeeringState,
 		client:             client,
 		provider:           provider,
-		remotePeeringName:  vpcPeeringState.ObjAsVpcPeering().Spec.VpcPeering.Gcp.RemotePeeringName,
-		remoteVpc:          vpcPeeringState.ObjAsVpcPeering().Spec.VpcPeering.Gcp.RemoteVpc,
-		remoteProject:      vpcPeeringState.ObjAsVpcPeering().Spec.VpcPeering.Gcp.RemoteProject,
-		importCustomRoutes: vpcPeeringState.ObjAsVpcPeering().Spec.VpcPeering.Gcp.ImportCustomRoutes,
+		remotePeeringName:  vpcPeeringState.ObjAsVpcPeering().Spec.Details.PeeringName,
+		importCustomRoutes: vpcPeeringState.ObjAsVpcPeering().Spec.Details.ImportCustomRoutes,
 	}
 }
 
 // Using cm- prefix to make it clear it's a cloud manager resource
 // Using obj name suffix as the peering name since it is unique within the kcp namespace
 func (s *State) getKymaVpcPeeringName() string {
-	return "cm-" + s.Obj().GetName()
+	return "cm--" + s.Obj().GetName()
```

Comment:
I think this change is not necessary. Or is there some specific reason for this change?